src/features/dataset/state/datasetState.ts

```diff
@@ -5,11 +5,18 @@ import Dataset, { NewDataset } from '../../../qri/dataset';
 import { qriRefFromString } from '../../../qri/ref';
 import { RenameDatasetAction } from './datasetActions';
 import { selectSessionUser } from '../../session/state/sessionState';
+import { newVersionInfo, VersionInfo } from "../../../qri/versionInfo";
 
 export const RENAME_NEW_DATASET = 'RENAME_NEW_DATASET'
 
 export const selectDataset = (state: RootState): Dataset => state.dataset.dataset
 
+export const selectDatasetVersionInfo = (state: RootState): VersionInfo => state.dataset.datasetVersionInfo
+
+export const selectDatasetVersionCount = (state: RootState): number | undefined => state.dataset.datasetVersionInfo.commitCount
+
+export const selectDatasetLogCount = (state: RootState): number | undefined => state.dataset.datasetVersionInfo.commitCount
```

Comment:
this method seems to do the same thing as the above method. We should remove this and have any calls to this function be replaced by `selectCommitCount`